cmake/templates/oneDALConfig.cmake.in

```diff
@@ -121,7 +121,7 @@ elseif (ONEDAL_USE_DPCPP STREQUAL "yes" AND ONEDAL_INTERFACE STREQUAL "yes")
 endif()
 
 list(APPEND DAL_LIBS onedal_core${DAL_DEBUG_SUFFIX})
-if(ONEDAL_USE_PARAMETERS_LIBRARY)
+if(ONEDAL_USE_PARAMETERS_LIBRARY AND ONEDAL_INTERFACE STREQUAL "yes")
```

Comment:
I think we can move  this entire section to elseif section above ?
elseif (ONEDAL_USE_DPCPP STREQUAL "no" AND ONEDAL_INTERFACE STREQUAL "yes")

Reply:
I'll swap it and see if I can run the dpc examples on their own without onedal_parameters

Reply:
the main idea is to have single if group rather than multiple through the file